packages/back-end/src/models/MetricModel.ts

```diff
@@ -170,6 +170,14 @@ export async function deleteMetricById(
   if (metric.managedBy === "api" && context.auditUser?.type !== "api_key") {
     throw new Error("Cannot delete a metric managed by the API");
   }
+  if (
+    metric.managedBy === "admin" &&
+    !context.permissions.canDeleteOfficialResources({
+      projects: metric.projects,
+    })
+  ) {
+    throw new Error("Cannot delete an official metric");
```

Comment:
Either throw the built-in permission error or change this language to be `You do not have permission to ...`.  Otherwise, the error is not clear.  Same for updating below.

Reply:
Whoops.